File: coding_round2_struct/Two_Sum/starter_code/solution.java
import java.util.*;
public class Main {
    public static void main(String[] args){
        Scanner sc = new Scanner(System.in);
        if(!sc.hasNextInt()) return;
        int n = sc.nextInt();
        long[] a = new long[n];
        for(int i=0;i<n;i++) a[i]=sc.nextLong();
        long target = sc.nextLong();
        // TODO: find pair of indices and print "i j"
    }
}

File: coding_round2_struct/Two_Sum/starter_code/solution.c
#include <stdio.h>
#include <stdlib.h>
int main(){
    int n; if(scanf("%d", &n)!=1) return 0;
    long long* a = malloc(sizeof(long long)*n);
    for(int i=0;i<n;i++) scanf("%lld", &a[i]);
    long long target; scanf("%lld", &target);
    // TODO: find pair of indices and print "i j"
    free(a);
    return 0;
}
